Neos.Flow/Classes/Security/Authentication/AuthenticationProviderManager.php

```diff
@@ -377,5 +395,7 @@ protected function buildProvidersAndTokensFromConfiguration(array $providerConfi
                 $tokenInstance->setAuthenticationEntryPoint($entryPoint);
             }
         }
+
+        $this->providerConfigurations = null;
```

Comment:
So `buildProvidersAndTokensFromConfiguration()` sets `$this->providerConfigurations` to `null`? That's kind of twisted..
Couldn't this be changed to sth like
```php
protected function buildProvidersAndTokensFromConfiguration()
{
    if ($this->tokens !== null) {
        return;
    }
    // ...
}
```

(and don't initialize `$tokens` at declaration time)

Reply:
Will have a look, could work.